packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -588,6 +588,40 @@ export default class TabAdvanced extends Component {
     blockedOperations => STOP_OPERATIONS.every(op => op in blockedOperations)
   )
 
+  _getIsMigrationAllowed = createSelector(
```

Comment:
Revert condition to make it more in line with other operations.

Reply:
Done